catalog/spatial/geocoding/spatial-geocoding-create/src/main/java/org/codice/ddf/spatial/geocoding/create/GeoNamesCreator.java

```diff
@@ -33,6 +33,7 @@ public GeoEntry createGeoEntry(final String line) {
                 .featureCode(fields[7])
                 .population(Long.parseLong(fields[14]))
                 .alternateNames(fields[3])
+                .countryCode(fields[8])
```

Comment:
I will wrap this in a try catch. I also think that the numbers are self explanatory based on the methods they are being passed into.